.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I don't think this file needs to be committed.

Reply:
fixed

Reply:
fixed in https://github.com/k1LoW/tbls/pull/383/commits/10e897eda3d5f9ff9abebcad264d899bbb6decd9